README.md

```diff
@@ -1,46 +1,28 @@
 # highlightjs-curl
 
-Support for using `highlight.js` to syntax highlight cURL commands. See https://highlightjs.org/ for more information about highlight.js.
-
-See https://curl.haxx.se/docs/manpage.html or in your shell use `curl --help` for more information about cURL.
+Support for using `highlight.js` to syntax highlight cURL commands. See https://highlightjs.org/ for more information about highlight.js. See https://curl.haxx.se/docs/manpage.html or in your shell use `curl --help` for more information about cURL.
 
 ## Usage
 
-Include the `highlight.js` script package in your webpage or node app, load this module and register it with `hljs`. Follow instructions at [highlightjs](https://highlightjs.org/) to learn how to include the library and CSS.
-
-If you're not using a build system and just want to embed this in your webpage:
-
-```html
-<script type="text/javascript" src="/path/to/highlight.pack.js"></script>
-<script type="text/javascript" src="/path/to/highlightjs-curl/curl.js"></script>
-<script type="text/javascript">
-    hljs.registerLanguage('curl', window.hljsDefineCurl);
-    hljs.initHighlightingOnLoad();
-</script>
-```
+Include the `highlight.js` script package in your webpage or node app, load this module and register it with `hljs`. Follow instructions at [highlightjs](https://highlightjs.org/usage/) to learn how to include the library and CSS. See [Getting started](https://github.com/highlightjs/highlight.js#getting-started) for different integration and module options.
 
-If you're using webpack / rollup / browserify / node:
+This cURL module is not part of the standard distribution and must be loaded separately. The module name is `curl.min.js` or `curl`, depending on how you reference the module from your bundler code.
 
-```javascript
-var hljs = require('highlightjs');
-var hljsDefineCUrl = require('highlightjs-curl');
+Once loaded, mark the code you want to highlight with the `curl` class or `language-curl`:
```

Comment:
```suggestion
Once loaded, mark the code you want to highlight with the `language-curl`:
```

We're going to start deprecating the short form and it may go away eventually.